### PR TITLE
Add TURN permissions mechanism

### PR DESCRIPTION
**Pull request by arkgil**
Addresses #76 

Added one method to `Jerboa.Client`: `create_permission`. It issues CreatePermission request to the server, which creates permission or refreshes existing one.

### CODE REVIEW

**Comment by arkgil** [comment]
@rslota @erszcz could you take a look at it? I'm blocked by this one.

File: lib/jerboa/client.ex
Comment (rslota) [inline lib/jerboa/client.ex:134]: There will be no API for refreshing several peers at once?

Comment (rslota) [reply lib/jerboa/client.ex:134]: I think we could add one, since it doesn't hurt us at all and we can leave `create_permission/2` that doesn't require list as argument as override.

Comment (arkgil) [reply lib/jerboa/client.ex:134]: We can't handle multiple permissions yet because of `Params` module. `put_atrr` will override attribute if such attribute is already present in struct, and `get_attr` will return only first occurence of attribute.

Comment (arkgil) [reply lib/jerboa/client.ex:134]: See #87

Comment (rslota) [reply lib/jerboa/client.ex:134]: Oh, I see... Well, server assumes that it can iterate though params to match and fetch multiple peers. And the standard says, that it `MUST` be possible. Therefore I think that we need to at least add task for extending encoder/decoder to our backlog. I don't mind that `Params.get_attr/2` returns first occurrence of attribute. That's fine. The most important thing is that decoder has to return multiple peer attributes and doesn't skip anything. 

If the client part - encoding - is too much work, then we may skip it for now, but we should have a ticket for that IMHO.

File: lib/jerboa/client.ex
Comment (rslota) [inline lib/jerboa/client.ex:68]: Why do we need sigil here?

File: lib/jerboa/client.ex
Comment (arkgil) [reply lib/jerboa/client.ex:134]: It'll be a quick fix, only `Params.get_attr/put_attr` has to change.

File: lib/jerboa/client.ex
Comment (arkgil) [reply lib/jerboa/client.ex:68]: Credo... It complains that there are multiple quote characters inside a string, so it's better to use `~S` sigil which creates a raw string. Credo can't understand that it's a docstring which already is a raw string..

File: lib/jerboa/client.ex
Comment (rslota) [inline lib/jerboa/client.ex:134]: Does the RFC states that we should only add peer permissions based on only IP address?

Comment (arkgil) [reply lib/jerboa/client.ex:134]: Yes, port does not matter. I left a comment about that in esl/fennec#25

**Review by rslota — APPROVED** [review]
Code looks good :)

**Comment by rslota** [comment]
I don't have write access to this repo, so I can't merge it.

**Comment by arkgil** [comment]
You do now :)

**Pull request merged**
